py_zipkin/encoding/protobuf/__init__.py

```diff
@@ -0,0 +1,167 @@
+# -*- coding: utf-8 -*-
+import socket
+import struct
+
+from py_zipkin.encoding._types import Kind
+from py_zipkin.util import unsigned_hex_to_signed_int
+try:
+    from py_zipkin.encoding.protobuf import zipkin_pb2
+except ImportError:  # pragma: no cover
+    zipkin_pb2 = None
```

Comment:
this is the small confusion I perceive. proto3 exists and this makes me think.. is this proto2 encoding?

Confusion may only exist in my head

Reply:
Yeah this is very confusing. https://github.com/grpc/grpc/issues/15444 explains the 2 suffix, which has nothing to do with the protobuf version.